packages/asc-ui/src/components/Button/ButtonStyle.ts

```diff
@@ -29,10 +32,14 @@ const ButtonStyle = styled(ButtonBaseStyle)<Props>`
       `}
   }
 
-  & > svg {
-    width: 30px;
-    height: 30px;
-    ${({ theme }) => svgFill('tint', 'level1')({ theme })};
+  /* stylelint-disable-next-line selector-type-no-unknown */
+  & > ${IconStyle}:first-child {
```

Comment:
Doesn't `& > ${IconStyle}:first-of-type {` resolve the `stylelint` issue

Reply:
fixed